__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should check TARGET_MIGRATION_VERSION to 7', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+  it('TEST should check TARGET_MIGRATION_VERSION to 8', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case is updated to check if `TARGET_MIGRATION_VERSION` is 8 instead of 7. This change seems to be in line with the database migration that has been performed, as typically the `TARGET_MIGRATION_VERSION` would be incremented after a new migration. However, it's important to ensure that the actual migration script increments the version to 8 as well.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->